src/util.spec.ts

```diff
@@ -20,11 +20,21 @@
 
 import * as extensionApi from '@podman-desktop/api';
 import { afterEach, beforeEach, expect, test, vi } from 'vitest';
-import { detectMinikube, getMinikubePath, installBinaryToSystem } from './util';
+import { detectMinikube, getMinikubePath, getMinikubeHome, installBinaryToSystem } from './util';
 import type { MinikubeInstaller } from './minikube-installer';
 import * as fs from 'node:fs';
 import * as path from 'node:path';
 
+const { config } = vi.hoisted(() => {
+  return {
+    config: {
+      get: vi.fn(),
+      has: vi.fn(),
+      update: vi.fn(),
+    },
+  };
+});
+
```

Comment:
we don't need the hoisted block, you can return mock (vi.fn) directly in getConfiguration method 

using the vi.mockImplementation we have typechecking while here we don't

Reply:
I've tried, but not figured out, how to make this return different value on different tests. I tried to do like other tests in this project and podman-desktop project, but without success 😞 

a little help, like example would be awesome